airbyte-cdk/python/airbyte_cdk/sources/declarative/connector_manifest_schema_openapi.json

```diff
@@ -0,0 +1,1493 @@
+{
```

Comment:
did you generate this from config_component_schema.json?

Reply:
Yeah this was copied over from that file and then modified a bit to convert it into an "openapi spec" type file. This is only being done right now as a workaround due to orval (our openapi typescript-generator library) still having issues with importing external json schemas. This workaround lets me just treat this entire file as its own openapi spec so I can [use orval to generate typescript directly from it](https://github.com/airbytehq/airbyte/pull/19731/files#diff-748ed30ef7eb13e66c7f64efcff9462812e56f3c50247731b613acb3a9cb10f9).

But I am still hopeful that orval will be properly fixed so that this isn't necessary, as the maintainer has been going back and forth with me and making incremental progress towards the right solution